README.md

```diff
@@ -1,4 +1,6 @@
-# Haveno
+<div align="center"> 
+  <img src="https://raw.githubusercontent.com/haveno-dex/haveno-meta/721e52919b28b44d12b6e1e5dac57265f1c05cda/logo/haveno_logo_landscape.svg" alt="Haveno logo">
+</div>
 
```

Comment:
(`utxobr` here) we keep getting questions around "how to pronounce it", how about adding the phonetics of it?

```
PA(key): /haˈveno/
Hyphenation: ha‧ve‧no
```

_(^ see https://en.wiktionary.org/wiki/haveno)_

example: https://github.com/vmware-tanzu/carvel-kbld indicates how to pronounce it.

Reply:
Sure! Consider that i'm currently writing a more detailed FAQ, so most info will go there, but makes sense to briefly mention the pronunciation in the README.

Added to "What is Haveno?": `Haveno (pronounced ha‧ve‧no) is a private and decentralized way...`

Reply:
sweet!